### PR TITLE
Add win64 build.

### PR DESCRIPTION
**Pull request by swenger**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
`jemalloc` can be ignored on Windows. This is a unix-only library. Thus a `# [not win]` selector would be the right thing for that line.

**Comment by xhochy** [comment]
@swenger you need to adjust the meta.yml. The failure is already on the conda side, not in the build script (scope).

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      -DWITH_BENCHMARK_TOOLS:BOOL=OFF ^
+      -DWITH_GFLAGS=ON ^
+      -DWITH_LZ4=ON ^
+      -DWITH_SNAPPY=ON ^
+      -DWITH_TESTS=OFF ^
+      -DWITH_TOOLS=OFF ^
+      -DWITH_ZLIB=ON ^
+      -DWITH_ZSTD=ON ^
+      -WITH_BZ2=ON ^
+      -WITH_JEMALLOC=OFF ^
```
Comment (xhochy) [inline recipe/bld.bat:25]:
```suggestion
      -WITH_JEMALLOC=OFF ^
      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,9 +22,10 @@ requirements:
     - {{ compiler("cxx") }}
     - cmake >=3.5.1,<4
     - ninja >=1.10,<2
+    - conda-verify
```
Comment (xhochy) [inline recipe/meta.yaml]: This is deprecated and shall not be used.
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -34,9 +34,11 @@ requirements:
...
   commands:
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
+    - test -f ${PREFIX}/lib/librocksdb.a        # [not win]
+    - test -f ${PREFIX}/lib/rocksdb.lib         # [win]
+    - test -f ${PREFIX}/lib/librocksdb.dylib    # [osx]
+    - test -f ${PREFIX}/lib/librocksdb.so       # [linux]
+    - test -f ${PREFIX}/lib/rocksdb-shared.lib  # [win]
     - test -f ${PREFIX}/include/rocksdb/db.h
```
Comment (xhochy) [inline recipe/meta.yaml]: Windows needs CMD syntax
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                   # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}         # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h               # [unix]
    - if not exist %LIBRARY_PREFIX\lib\rocksdb-shared.lib  # [win]
    - if not exist %LIBRARY_PREFIX\include\rocksdb\db.h    # [win]
    - if not exist %LIBRARY_PREFIX%\librocksdb.lib         # [win]
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -34,10 +33,12 @@ requirements:
...
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
-    - test -f ${PREFIX}/include/rocksdb/db.h
+    - test -f ${PREFIX}/lib/librocksdb.a                   # [unix]
+    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}         # [unix]
+    - test -f ${PREFIX}/include/rocksdb/db.h               # [unix]
+    - if not exist %LIBRARY_PREFIX\lib\rocksdb-shared.lib  # [win]
+    - if not exist %LIBRARY_PREFIX\include\rocksdb\db.h    # [win]
+    - if not exist %LIBRARY_PREFIX%\librocksdb.lib         # [win]
```
Comment (swenger) [inline recipe/meta.yaml]:
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                    # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}          # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h                # [unix]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib  # [win]
    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h    # [win]
    - if not exist %LIBRARY_PREFIX%\librocksdb.lib          # [win]
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -34,10 +33,12 @@ requirements:
...
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
-    - test -f ${PREFIX}/include/rocksdb/db.h
+    - test -f ${PREFIX}/lib/librocksdb.a                    # [unix]
+    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}          # [unix]
+    - test -f ${PREFIX}/include/rocksdb/db.h                # [unix]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
+    - if not exist %LIBRARY_PREFIX%\librocksdb.lib exit 1          # [win]
```
Comment (swenger) [inline recipe/meta.yaml]:
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                            # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}                  # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h                        # [unix]
    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1     # [win]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1   # [win]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1          # [win]
```

File: recipe/meta.yaml [hunk 6]
```diff
@@ -34,10 +33,12 @@ requirements:
...
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
-    - test -f ${PREFIX}/include/rocksdb/db.h
+    - test -f ${PREFIX}/lib/librocksdb.a                    # [unix]
+    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}          # [unix]
+    - test -f ${PREFIX}/include/rocksdb/db.h                # [unix]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
+    - if not exist %LIBRARY_PREFIX%\rocksdb.lib exit 1             # [win]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1             # [win]
```

File: recipe/meta.yaml [hunk 7]
```diff
@@ -34,10 +33,12 @@ requirements:
...
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
-    - test -f ${PREFIX}/include/rocksdb/db.h
+    - test -f ${PREFIX}/lib/librocksdb.a                    # [unix]
+    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}          # [unix]
+    - test -f ${PREFIX}/include/rocksdb/db.h                # [unix]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1             # [win]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1         # [win]
```

File: recipe/meta.yaml (hunk 6 above)
Comment (swenger) [inline recipe/meta.yaml]:
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                           # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}                 # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h                       # [unix]
    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1         # [win]
```

File: recipe/meta.yaml [hunk 8]
```diff
@@ -34,10 +33,12 @@ requirements:
...
-    - test -f ${PREFIX}/lib/librocksdb.a
-    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
-    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
-    - test -f ${PREFIX}/include/rocksdb/db.h
+    - test -f ${PREFIX}/lib/librocksdb.a                    # [unix]
+    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}          # [unix]
+    - test -f ${PREFIX}/include/rocksdb/db.h                # [unix]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
+    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1         # [win]
```
Comment (swenger) [inline recipe/meta.yaml]:
```suggestion
    - test -f ${PREFIX}/lib/librocksdb.a                           # [unix]
    - test -f ${PREFIX}/lib/librocksdb${SHLIB_EXT}                 # [unix]
    - test -f ${PREFIX}/include/rocksdb/db.h                       # [unix]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb.lib exit 1         # [win]
    - if not exist %LIBRARY_PREFIX%\lib\rocksdb-shared.lib exit 1  # [win]
    - if not exist %LIBRARY_PREFIX%\include\rocksdb\db.h exit 1    # [win]
```

File: recipe/bld.bat [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+echo Running CMake build...
+cmake --build build
+if errorlevel 1 exit 1
+
+echo Running CMake install...
+cmake --install build
+if errorlevel 1 exit 1
+
+echo Contents of output directory %LIBRARY_PREFIX%:
+dir /S %LIBRARY_PREFIX%
```
Comment (xhochy) [inline recipe/bld.bat]:
```suggestion
```

**Review by xhochy — COMMENTED** [review]
Small cleanup and then this should be good to go

**Comment by swenger** [comment]
Thanks Uwe!

**Comment by xhochy** [comment]
@conda-forge/rocksdb This is ready for review/merge.

**Review by hadim — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**